orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/tasks/manifests/CreateBakeManifestTask.java

```diff
@@ -96,12 +99,21 @@ public TaskResult execute(@Nonnull Stage stage) {
 
     String outputArtifactName = expectedArtifacts.get(0).getMatchArtifact().getName();
 
+    Map<String, Object> overrides = context.getOverrides();
+    if (context.getEvaluateOverrideExpressions()) {
```

Comment:
We'll still need a null check here, as `if(null)` is an NPE; alternately we could change the member variable on `BakeManifestContext` to a `boolean` and handle the null in the constructor.  (In this case, setting `false` if null, in the `Deploy` case setting `true`.)

Reply:
Thanks for the wonderful code review and java schooling @ezimanyi! Just re-tested and added null checks to each class, so this should be ready for one final look :pray: